src/routes/favorites/FavoritesScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import { RouteProp, useFocusEffect } from '@react-navigation/native'
```

Comment:
Not having a folder structur is really bad practice, but I see your point though. I think it would make sense to put every route in a subfolder also it they do no have components yet.

Reply:
i agree with Steffi in this case. We may add a guideline for naming & structuring of components/routes and stick to them. Sure once you move it its a pain to review but after having a clear structure this shouldn't happen. I also support structures like `baseComponents` Text/Buttons/TextInput etc. I think finding components and routes in integreat is quite a pain